test/integrations/destinations/facebook_pixel/processor/ecommTestData.ts

```diff
@@ -0,0 +1,705 @@
+import { VERSION } from '../../../../../src/v0/destinations/facebook_pixel/config';
+import { generateTrackPayload, generateMetadata, transformResultBuilder } from '../../../testUtils';
+import { Destination } from '../../../../../src/types';
+import { ProcessorTestData } from '../../../testTypes';
+
+const commonDestination: Destination = {
+  ID: '12335',
+  Name: 'sample-destination',
+  DestinationDefinition: {
+    ID: '123',
+    Name: 'facebook_pixel',
+    DisplayName: 'Facebook Pixel',
+    Config: {},
+  },
+  WorkspaceID: '123',
+  Transformations: [],
+  Config: {
+    blacklistPiiProperties: [
+      {
+        blacklistPiiProperties: '',
+        blacklistPiiHash: false,
+      },
+    ],
+    accessToken: '09876',
+    pixelId: 'dummyPixelId',
+    eventsToEvents: [
+      {
+        from: 'ABC Started',
+        to: 'InitiateCheckout',
+      },
+    ],
+    eventCustomProperties: [
+      {
+        eventCustomProperties: '',
+      },
+    ],
+    valueFieldIdentifier: '',
+    advancedMapping: false,
+    whitelistPiiProperties: [
+      {
+        whitelistPiiProperties: 'email',
+      },
+    ],
+  },
+  Enabled: true,
+};
+
+const commonUserTraits = {
+  email: 'abc@gmail.com',
+  anonymousId: 'c82cbdff-e5be-4009-ac78-cdeea09ab4b1',
+  event_id: '12345',
+};
+
+const commonUserProperties = {
+  revenue: 400,
+  additional_bet_index: 0,
+  email: 'abc@gmail.com',
+};
+
+const commonPropertiesWithoutProductArray = {
+  category: 'dummy',
+  quantity: 10,
+  value: 100,
+  product_id: '12345',
+};
+
+const commonPropertiesWithProductArray = {
+  products: [
+    {
+      product_id: '017c6f5d5cf86a4b22432066',
+      sku: '8732-98',
+      name: 'Just Another Game',
+      price: 22,
+      position: 2,
+      category: 'Games and Entertainment',
+      url: 'https://www.myecommercewebsite.com/product',
+      image_url: 'https://www.myecommercewebsite.com/product/path.jpg',
+    },
+    {
+      product_id: '89ac6f5d5cf86a4b64eac145',
+      sku: '1267-01',
+      name: 'Wrestling Trump Cards',
+      price: 4,
+      position: 21,
+      category: 'Card Games',
+    },
+  ],
+  category: 'dummy',
+  quantity: 10,
+  revenue: 100,
+  price: 50,
+  product_id: '12345',
+  order_id: '23456',
+};
+const commonTimestamp = new Date('2023-10-14');
+const commonStatTags = {
+  errorCategory: 'dataValidation',
+  errorType: 'instrumentation',
+  destType: 'FACEBOOK_PIXEL',
+  module: 'destination',
+  implementation: 'native',
+  feature: 'processor',
+};
+
+export const ecommTestData: ProcessorTestData[] = [
+  {
+    id: 'facebook_pixel-test-1',
```

Comment:
Make sure every test has unique Id, for all the fb_pixel test data

Reply:
updated